main.md

```diff
@@ -234,6 +234,12 @@ holder and issuer MAY use pre-established key material.
 
 Note: need to define how holder public key is included, right now examples are using `sub_jwk` I think.
 
+#### Hash of an SVC
+
+* `svc_hash`
+
+  * REQUIRED. SVC hash value. Its value is the base64url encoding of the left-most half of the hash of the octets of the ASCII representation of the SVC value, where the hash algorithm used is the same hash algorithm that is used for the selectively disclosable claims in an SD-JWT. The svc_hash value is a case sensitive string.
```

Comment:
I have questions about "the ASCII representation of the SVC value".  Is this representation the entire SVC, such as in the example at https://www.ietf.org/archive/id/draft-fett-oauth-selective-disclosure-jwt-00.html#section-4.2.2 ?  If so, ASCII should almost certainly be replaced by UTF-8, since JSON can contain non-ASCII characters.  If it's something else, then we also likely need to be more descriptive about what it is.

Reply:
UTF-8 contains ASCII, this latter is a subset of the first.
The hashes are hexadecimal values in ascii format.

We may say that the text is good as it is if I'm not missing something @selfissued :)